tests/sentry/issues/auto_source_code_config/test_process_event.py

```diff
@@ -98,10 +103,12 @@ def create_repo_and_code_mapping(
             project_id=self.project.id,
             stack_root=stack_root,
             source_root=source_root,
+            default_branch=default_branch,
             repository=repository,
             organization_integration_id=organization_integration.id,
             integration_id=organization_integration.integration_id,
             organization_id=organization_integration.organization_id,
+            automatically_generated=automatically_generated,
```

Comment:
This was needed for #89859 (which has been reverted), but it's good to have it around.